ERROR - 2025-05-08 07:17:48 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:17:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:17:52 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:17:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:17:58 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:17:58 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:01 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:01 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:04 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:04 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:12 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:12 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:14 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:14 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:20 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:24 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:25 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-05-08 07:18:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: user]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `deleted_at` IS NULL
AND `tb_dashboard`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:26 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Timsupport.php(12): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Timsupport->index()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Timsupport))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Timsupport.php(12): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Timsupport->index()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Timsupport))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\itsupportpln\app\Controllers\Timsupport.php(12): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Timsupport->index()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Timsupport))
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-05-08 07:18:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: timsupport]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Timsupport.php(12): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Timsupport->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Timsupport))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:34 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#7 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#8 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#11 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:18:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:18:35 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-05-08 07:18:35 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: user]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `deleted_at` IS NULL
AND `tb_dashboard`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:19:24 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#6 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\itsupportpln\app\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\User->index()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-05-08 07:19:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: php_network_getaddresses: getaddrinfo for http failed: No such host is known. 
[Method: GET, Route: user]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `deleted_at` IS NULL
AND `tb_dashboard`.`deleted_at` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\User.php(21): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\User->index()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:20:09 --> Error connecting to the database: ErrorException: mysqli::real_connect(): php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'C:\\xampp\\htdocs...', 201)
#1 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#5 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#6 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#7 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#8 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#9 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#10 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#13 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#14 {main}

Next mysqli_sql_exception: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('http://asia-eas...', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: php_network_getaddresses: getaddrinfo for http failed: No such host is known.  in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2025-05-08 07:21:23 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'tahr'@'127.0.0.6' (using password: YES) in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('asia-east1-001....', 'tahr', Object(SensitiveParameterValue), 'decisive-amaran...', 30698, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'127.0.0.6' (using password: YES) in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2025-05-08 07:31:52 --> mysqli_sql_exception: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Auth->login()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:31:52 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist
[Method: POST, Route: auth/login]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-08 07:31:52 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-08 07:31:52 --> [Caused by] mysqli_sql_exception: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:32:51 --> mysqli_sql_exception: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Auth->login()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:32:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist
[Method: POST, Route: auth/login]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 2 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 4 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-08 07:32:51 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 4 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 6 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-05-08 07:32:51 --> [Caused by] mysqli_sql_exception: Table 'decisive-amaranth-donkey.tb_dashboard' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `tb_dashboard`
WHERE `Username` = \'admin\'
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 7 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:43:46 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'dashboard', 3306, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:43:46 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'dashboard'
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:43:51 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'dashboard', 3306, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:43:51 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'dashboard'
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:43:56 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'dashboard', 3306, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#10 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}
CRITICAL - 2025-05-08 07:43:56 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'dashboard'
[Method: GET, Route: dashboard/get-random-data]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `timsupport`
ORDER BY RAND()
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Models\DashboardModel.php(23): CodeIgniter\BaseModel->first()
 6 APPPATH\Controllers\Dashboard.php(51): App\Models\DashboardModel->getRandomTeam()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Dashboard->getRandomData()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Dashboard))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:44:25 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'dashboard', 3306, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\itsupportpln\app\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Auth->login()
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\itsupportpln\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\itsupportpln\system\BaseModel.php(713): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\itsupportpln\app\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(933): App\Controllers\Auth->login()
#7 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\itsupportpln\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\itsupportpln\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 {main}
CRITICAL - 2025-05-08 07:44:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'dashboard'
[Method: POST, Route: auth/login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `tb_dashboard`
WHERE `Username` = :Username:
AND `tb_dashboard`.`deleted_at` IS NULL
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(713): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(29): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->login()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-05-08 07:44:33 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:201
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php(201): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'dashboard', 3306, '', 0)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#4 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#5 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#6 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#8 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#9 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#11 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#12 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'dashboard' in C:\xampp\htdocs\itsupportpln\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1525): CodeIgniter\Database\BaseConnection->query('SHOW TABLES FRO...')
#3 C:\xampp\htdocs\itsupportpln\system\Database\BaseConnection.php(1545): CodeIgniter\Database\BaseConnection->listTables()
#4 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(758): CodeIgniter\Database\BaseConnection->tableExists('migrations')
#5 C:\xampp\htdocs\itsupportpln\system\Database\MigrationRunner.php(162): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 C:\xampp\htdocs\itsupportpln\system\Commands\Database\Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest(NULL)
#7 C:\xampp\htdocs\itsupportpln\system\CLI\Commands.php(70): CodeIgniter\Commands\Database\Migrate->run(Array)
#8 C:\xampp\htdocs\itsupportpln\system\CLI\Console.php(48): CodeIgniter\CLI\Commands->run('migrate', Array)
#9 C:\xampp\htdocs\itsupportpln\system\Boot.php(360): CodeIgniter\CLI\Console->run()
#10 C:\xampp\htdocs\itsupportpln\system\Boot.php(104): CodeIgniter\Boot::runCommand(Object(CodeIgniter\CLI\Console))
#11 C:\xampp\htdocs\itsupportpln\spark(84): CodeIgniter\Boot::bootSpark(Object(Config\Paths))
#12 {main}
ERROR - 2025-05-08 07:58:34 --> Email: Unable to send email using SMTP. Your server might not be configured to send mail using this method.
ERROR - 2025-05-08 07:58:34 --> 220 smtp.gmail.com ESMTP d9443c01a7336-22e66a973e9sm26522825ad.167 - gsmtp

hello: 250-smtp.gmail.com at your service, [180.254.70.11]
250-SIZE 35882577
250-8BITMIME
250-STARTTLS
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

starttls: 220 2.0.0 Ready to start TLS

hello: 250-smtp.gmail.com at your service, [180.254.70.11]
250-SIZE 35882577
250-8BITMIME
250-AUTH LOGIN PLAIN XOAUTH2 PLAIN-CLIENTTOKEN OAUTHBEARER XOAUTH
250-ENHANCEDSTATUSCODES
250-PIPELINING
250-CHUNKING
250 SMTPUTF8

Failed to authenticate password. Error: 535-5.7.8 Username and Password not accepted. For more information, go to
535 5.7.8  https://support.google.com/mail/?p=BadCredentials d9443c01a7336-22e66a973e9sm26522825ad.167 - gsmtp

